datastore/longhorn.go

```diff
@@ -4881,3 +4881,14 @@ func IsBackendStoreDriverV1(backendstoreDriver longhorn.BackendStoreDriverType)
 func IsBackendStoreDriverV2(backendstoreDriver longhorn.BackendStoreDriverType) bool {
 	return backendstoreDriver == longhorn.BackendStoreDriverTypeV2
 }
+
+// IsFileSystemSupportedVolumeSize returns turn if the volume size is supported by the given fsType file system.
+func IsFileSystemSupportedVolumeSize(backendstoreDriver longhorn.BackendStoreDriverType, fsType string, volumeSize int64) bool {
+	if IsBackendStoreDriverV1(backendstoreDriver) {
+		// unix.Statfs can not differentiate the ext2/ext3/ext4 file systems.
+		if (strings.HasPrefix(fsType, "ext") && volumeSize >= util.MaxExt4VolumeSize) || (fsType == "xfs" && volumeSize >= util.MaxXfsVolumeSize) {
+			return false
+		}
+	}
```

Comment:
Two points

1. Rename `IsFileSystemSupportedVolumeSize` to `IsSupportedVolumeSize`
2.  Can you add 
```
TODO: check the logical volume maximum size limit
```

Reply:
Renamed the function and added the TODO comment.
Thanks.